### ISSUE TITLE
[Strategic Focus] Interoperability 

### ISSUE THREAD

**Issue by BenJam** [opencollective/opencollective#5488]
_Interoperability is about expanding our impact, enabling users to choose the tools that work for them and enabling us to focus on what we provide that others do not._ 

As a small team, we cannot tackle every problem. We are part of a broader movement of organisations, tools and technologies that are complementary to our own mission and goals. We should build for a future that enables us to share in each others' success. We accept that there is a tension between offering choice and configurability vs. consolidation and simplification and we will continue to balance the needs of users who exist on both sides of this divide. 

Key projects for this area include

- #1660 

If we're successful we should see

- An increase in financial support for Collectives
- An increase in engagement on Open Collective

**Comment by alanna** [comment]
Ideas for metrics:
* At least one new active integration between an outside tool and OC, led by the OC team (active means real people really using it), or a significant improvement to an existing one, e.g. we've talked about accounting software, third party event tools, next level of Github Sponsors integration, automation of our Benevity or Justworks processes, etc.
* At least one new active integration between an outside tool and OC, led by an outside team (e.g. some ideas floating around like Metagov, Cobudget, Crypto wallet).
* @znarf 's personal assessment of our API goes from "no comment" to at least "not bad".
* Time spent by the ops team manually processing interactions with outside tool goes down meaningfully.

Is API documentation included in this space? Because it's something we set aside and specifically wanted to revisit when integrations were more a priority. If so, then improved API documentation could be a measure of success.
